claim/validations.py

```diff
@@ -223,12 +223,12 @@ def validate_claimitem_in_price_list(claim, claimitem):
     target_date = get_claim_target_date(claim)
     pricelist_detail_qs = ItemsPricelistDetail.objects \
         .filter(item_id=claimitem.item_id,
-                validity_to__isnull=True,
+                validity_to__gte=target_date,
+                validity_from__lte=target_date,
```

Comment:
please use *filter_validity(validity=target_date)
```suggestion
                 *filter_validity(validity=target_date)
```